### ISSUE TITLE
Treat the wood of the Dura counter-tops

### ISSUE THREAD

**Issue by MetaFight** [OxfordHackspace/HelperWork#2]
The hackspace has acquired some nice furniture recently.  Some of these items have some nice wood counter-tops which would benefit from a bit of protection.

I'm no pro in this area, but I suspect treating the wood is the way to go.

This task would involve 3 phases:
1. Investigate the best way to protect these countertops (balancing cost, protection, and whatever else makes sense)
2. Run the plan by the hackspace admin
3. Carry out the plan

It's worth noting that it may make sense the create a separate task for item 3.

**Comment by Mellen** [comment]
I have sent a message to Dura, requesting information.

**Comment by Mellen** [comment]
Dura don't know anything.

>**(Dura)**  Thank you for your enquiry, however could you tell me when you purchased the worktops from us as I cannot find you on our ordering system.

>**(Me)** I don't have an order number. It was just a general question about maintaining the beechwood worktops.

> **(Dura)** Thank you for your reply, if you have purchased them from ourselves I can check with our supplier but I wouldn’t have the information if they are not.

**Comment by Mellen** [comment]
Dura have replied that the tops are treated with linseed oil. So we don't need to retreat them, but we can if we want.

**Comment by MetaFight** [comment]
Cool, thanks.

I think we may have some of the sample or display models which may have not been treated.  Lauren (@Yanaba) , can you confirm?

If that is the case, I guess treating them with linseed oil is the way to go.

**Comment by Yanaba** [comment]
Yes, we were told ours are not treated (except for one maybe iirc?) because ours were never prepped for selling. 

So ordering linseed oil sounds like the way to go. 

Next step is finding out the surface area we need to treat (add all worktop surface areas together). 

Then I guess find out how one applies it, if there's anything special to do other than rub it on with gloves. Maybe there is a sponge tool or something so we don't get darker areas etc, I don't know. 

Finally, shoot me a link for a price-shopped bottle of linseed oil that is made for this purpose and big enough to cover this surface area. Also for sponge etc if that applies. 

I'll buy it. 

Cheers guys.

**Comment by Mellen** [comment]
From http://www.diydoctor.org.uk/projects/treatwtops.htm

> Hardwood worktops must be thoroughly treated with at least three coats of protective wood oil, to all sides (including the underside), prior to installation. It is also beneficial to apply as many coats as possible to the end-grain of the worktops as this is the area which will absorb the most moisture. An extra three to four coats should also be applied around tapholes, sink cutouts and areas of heavy use or exposure.
> 
> Oils are best applied by using a lint-free cloth rather than a brush. A lint-free cloth is any form of material that will not deposit any particles on the wood - an old T-shirt is usually ideal.
> 
> When applying the oil yourself, it is best to apply several thin coats rather than one thick application. Always work the oil into the worktop in the direction of the wood grain for best results.
> After applying the first coat, you will probably notice that some areas are wet and others dry. This is because the wood will absorb the oil at different degrees in different areas. Wait for 15minutes and then run the cloth over the entire surface again to ensure an even spread. Do not allow large pools of oil to dry on the surface.
> 
> Apply the second and third coats in the same manner, allowing ½ hour - 1hour for drying time in between coats. If the surface feels slightly rough in certain areas, or if you wish to develop a smoother surface, then we recommend that you apply a further coat of oil using either a green nylon scouring sponge or wire wool.

**Comment by Mellen** [comment]
The central table is (mm) 1500 &times; 740 &times; 40
The other surfaces are (mm) 1000 &times; 566 &times; 40

**Comment by Yanaba** [comment]
Cool, how many surfaces?  We'll need to multiply to get surface area,
especially if we need enough for like 3 coats of linseed oil like that link
says.  There are how many surfaces aside from the table...6? 7?

**Comment by Mellen** [comment]
I have some [expert advice](http://diy.stackexchange.com/a/90499/53892) that the amount of linseed oil per unit surface area will be explained on the bottle. So first we should choose a brand.

**Comment by Mellen** [comment]
The area of the 5 tops 3.374m², but that should be doubled for the bottoms, so 6.748m². Then we have to do the edges. That's an extra 0.68032m², so a total of 7.42832m². According to the instructions I linked to above:

> Your new worktop should be treated with oil every week for the first six weeks, and at least once every three months thereafter.

So we'll need that much oil &times; six, plus extra for the tri-monthly treatments, which we might choose to buy later.

**Comment by Yanaba** [comment]
Sounds like a lot of work.

Ok, so that's for all the surfaces (cabinet tops + table) in the old
electronics room and also the one in metalworking (roller cabinet), yes? Is
the one in metalworking counted? I thought there were more than 5.

Thanks for all this btw.

**Comment by Mellen** [comment]
I was only aware of the counter tops in the electronics room.

**Comment by Mellen** [comment]
There isn't a beechwood top on the draws in the metalworking room.

**Comment by Mellen** [comment]
For Barretine boiled linseed oil (boiled is recommended for treating worktops) we would need a little over 6 litres for the full six week course (you can see the calculation for 3 coats [here](http://www.wood-finishes-direct.com/product/boiled-linseed-oil)). 5L costs £26.99 at that online store. 

According to the [Wickes store](http://www.wickes.co.uk/Wickes-Boiled-Linseed-Oil-500ml/p/600346?CAWELAID=120135120000004855&CAGPSPN=pla&CAAGID=17408411932&CATCI=pla-193347849412&utm_source=google&utm_medium=cpc&utm_term=&utm_campaign=Google%20Shopping%20-%20Garden&tmcampid=&tmad=c&tmplaceref=BHt9Se8P&tmcustom=mkwid|sBHt9Se8P_dc|pcrid|43364563852|kword||match||plid|&gclid=CJnq0JC918wCFSQW0wodBKQBdw), their 500ml covers 2m², so we'd need 12 bottles for 3 complete coats, 72 bottles for the 6 week course. That would cost £359.28.

[Liberon 500ml](http://www.axminster.co.uk/liberon-boiled-linseed-oil-500ml-340988?gclid=COrxwK7P18wCFTUo0wodz6cOnw) covers between 8m² and 12m², so (using the lower end) we'd need a maximum of 18 bottles for the six week course, at £5.69 each, totalling £102.42

These are the three I found that have coverage details. I have asked for details of two others.

**Comment by Yanaba** [comment]
Great work. ;)

Yeesh, what a price difference.

The drawers in metalworking do indeed have an associated top, but I think
it got taken off to put the model on top of them as it wasn't screwed on.
I don't know if it's still against one of the walls in metalworking or in
electronics now, but regardless everything we got from Dura should have one
top each iirc.

As to the Barretine link..."Rags soaked in boiled linseed oil are liable to
self ignite."

WTF? Um, good to know! Let's make sure that whoever is doing the work is
well aware of that fact, not cool.

**Comment by Mellen** [comment]
Yes, if they're left to dry out they can set alight. We'll need a bucket of water to put them in between coats.

**Comment by Yanaba** [comment]
Ok, based on this I've ordered 2 x 5L Barretine boiled linseed oil (it says
it is food safe as well), and two packages of lint-free application cloths.

Cloths: http://www.wood-finishes-direct.com/product/cotton-sheets

That should give us options for coating things for the foreseeable future,
though I think a chemicals cabinet may be becoming a priority.

The order should arrive Tuesday.

I suggest we figure out where we're going to oil these and on what
supports...I believe we recently got in some black sawhorses, in the front
room, and they would seem ideal for this. However I'm also keen we have
some ventilation wherever we're planning to do this as it sounds like
there's going to be some smell involved...the high voltages room comes to
mind as it has a door and windows and no sources of particles, but whatever
people decide. Not sure the old electronics/metal rooms are good for this
as there may be bashing around in there over the next little bit.

Cheers. ;)

**Comment by Yanaba** [comment]
Hi all,

The linseed oil has arrived. Now we need to commandeer a room with a door to close and windows to open, get a process and get some surfaces oiled! 

I suggest the lift room (or better, the high voltage room--it's perfect, is this possible?) because other rooms at the end of the hall may have dusty conditions before we're done. 

I've asked Nathan to put the package on the electronics room table for now. Cloths should be in there I think. 

L

**Comment by MetaFight** [comment]
I don't mind making room in the high voltage room.  The servers are in there too though, so we need to see what Jamie thinks about it as well.

**Comment by Yanaba** [comment]
That might actually be a good thing because their heat will encourage drying, but yes good to ask.

**Comment by Ilidur** [comment]
Just to bring this one back from the dead. @jlokier Would it be alright to dry these in that room or do we need to take the servers out first?

**Comment by Yanaba** [comment]
I think we should use the metals room for this. The floor is the right kind, windows are present, and it's the only one that doesn't have people working in it daily right now. 

The old electronics room, the server back room and the small room all have people using them nearly every day these days. Not so for the metals room though, yet. 

Also, the server room is carpeted so not great for this.
